storage/remote/queue_manager.go

```diff
@@ -120,6 +153,16 @@ func newQueueManagerMetrics(r prometheus.Registerer) *queueManagerMetrics {
 		},
 		[]string{remoteName, endpoint},
 	)
+	m.sentMetadataDuration = prometheus.NewHistogramVec(
```

Comment:
Would it be better to use `sentBatchDuration` for metadata as well since they are the same payload and destination? Perhaps with a label to separate them while samples and metadata are still sent in separate requests?

Reply:
Wouldn't the same concept apply to the other metrics (in particular to the `bytes_sent` family)? I'm not opposed to the approach but would like to have consistency, I'd love to have either all of them using a particular label or none of them.

Reply:
Yes, it would apply to all of them, and we would want consistency. I could have been more clear, and didn't feel like commenting on all the new metrics :)

Reply:
While executing this change, I realised that the HELP/MetricName would often contradict that we're doing by including the word sample in its text e.g. `Duration of sample batch send calls to the remote storage.` or `retried_samples_total`.

Do you think it's fine to proceed as is? Maybe the extra-label will be enough for users but wanted to double-check. For the ones were modifying the help text is enough, I don't think it's too big of a problem but changing the metric name would certainly be a no-go as it would break existing dashboards.

Reply:
We have changed metric names/labels before. I think editing the help text is enough to something like `Duration of send calls to the remote storage.`

Reply:
Things like `retried_samples_total` should stick to just samples, a `retried_metadata_total` could make sense as well if you wanted. Mostly I wanted the metrics measuring outgoing http calls to be nested under one thing as they use the same message format and will hopefully be combined in the future.

Reply:
> Mostly I wanted the metrics measuring outgoing http calls to be nested under one thing 

This makes perfect sense. I have unified the duration and total bytes of the request using a "type" label (similarly to what we do in the WAL watcher with series and samples). The other three (`succeededMetadataTotal`, `failedMetadataTotal`, and `retriedMetadataTotal`) I have left as is.